Program.cs

```diff
@@ -1,7 +1,33 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+
+namespace Training {
+   internal class Program {
+      static void Main (string[] args) {
+         string[] words = { "glory", "floor", "ant", "victory", "aegilops", "access" };
+         wordsLength = new int[words.Length];
+         int index = 0;
+         foreach (string word in words) {
+            if (abecedarian (word, index))
+               Console.WriteLine ($"Abecedarian word :{word}");
+            index++;
+         }
+         if (maxIndex != -1)
+            Console.WriteLine ($"'{words[maxIndex]}' is the longest abecedarian word of given array.");
+         else Console.WriteLine ("No abecedarian word here.");
+      }
+      static bool abecedarian (string word, int index) {

```

Comment:
Method name is not as per C# coding style.

Reply:
Changed